server/src/test/java/org/elasticsearch/index/mapper/GeoPointFieldTypeTests.java

```diff
@@ -36,31 +40,50 @@ public void testFetchSourceValue() throws IOException {
         Map<String, Object> otherJsonPoint = Map.of("type", "Point", "coordinates", List.of(30.0, 50.0));
         String wktPoint = "POINT (42.0 27.1)";
         String otherWktPoint = "POINT (30.0 50.0)";
+        byte[] wkbPoint = WellKnownBinary.toWKB(new Point(42.0, 27.1), ByteOrder.LITTLE_ENDIAN);
```

Comment:
I noticed more use of LITTLE_ENDIAN in the code-base than BIG_ENDIAN, I presume this is because x86 it little-endian, and we favour that? Should we allow configuration of this?

Reply:
I don't think so, there is no benefit for the user and it complicate testing.